uk.co.compucorp.civicrm.hrcore/CRM/HRCore/String/TimeUnitApplier.php

```diff
@@ -111,7 +115,7 @@ private static function getHoursPart($value) {
    * be the whole value we wan't to apply the unit to, but only the decimal part,
    * that represents the minutes.
    *
-   * @param number $value
+   * @param int $value
```

Comment:
Shouldn't this be int|float also? or probably just float.

Reply:
Not really, as this is actually the remainder we get here https://github.com/civicrm/civihr/blob/ec929a5576cffaeb92e8dfaf4b3b09bbb6481c46/uk.co.compucorp.civicrm.hrcore/CRM/HRCore/String/TimeUnitApplier.php#L91